### PR TITLE
Welcome Page: Move from brave-ui to brave-core

### PR DESCRIPTION
**Pull request by cezaraugusto**
address https://github.com/brave/brave-browser/issues/2335
close https://github.com/brave/brave-browser/issues/5216

### Test Plan

```
# ensure Welcome is properly displayed
npm run storybook
# build is ok
npm run build
```

### CODE REVIEW

**Comment by petemill** [comment]
I'm going to add the 'imported from' prefix as some of these commits don't mention the welcome page

File: components/brave_welcome_ui/containers/screens/footerBox.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   SkipButton,
   FooterButton,
   Bullet
-} from 'brave-ui/features/welcome'
+} from '../../../../components/brave_welcome_ui/components'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/footerBox.tsx]: nit: exits UI folder then goes back in - can we replace `../../../../components/brave_welcome_ui` with `../../`?

File: components/brave_welcome_ui/containers/screens/rewardsBox.tsx [hunk 2]
```diff
@@ -5,10 +5,10 @@
 import * as React from 'react'
 
 // Feature-specific components
-import { Content, Title, Paragraph, PrimaryButton } from 'brave-ui/features/welcome'
+import { Content, Title, Paragraph, PrimaryButton } from '../../../../components/brave_welcome_ui/components'
 
 // Images
-import { WelcomeRewardsImage } from 'brave-ui/features/welcome/images'
+import { WelcomeRewardsImage } from '../../../../components/brave_welcome_ui/components/images'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/rewardsBox.tsx]: Same nit for paths

File: components/brave_welcome_ui/containers/screens/searchBox.tsx [hunk 3]
```diff
@@ -6,10 +6,10 @@
 import * as React from 'react'
 
 // Feature-specific components
-import { Content, Title, Paragraph, PrimaryButton, SelectGrid, SelectBox } from 'brave-ui/features/welcome'
+import { Content, Title, Paragraph, PrimaryButton, SelectGrid, SelectBox } from '../../../../components/brave_welcome_ui/components'
 
 // Images
-import { WelcomeSearchImage } from 'brave-ui/features/welcome/images'
+import { WelcomeSearchImage } from '../../../../components/brave_welcome_ui/components/images'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/searchBox.tsx]: Same nit for paths

File: components/brave_welcome_ui/containers/screens/shieldsBox.tsx [hunk 4]
```diff
@@ -5,13 +5,13 @@
...
 // Feature-specific components
-import { Content, Title, Paragraph } from 'brave-ui/features/welcome'
+import { Content, Title, Paragraph } from '../../../../components/brave_welcome_ui/components'
 
 // Utils
 import { getLocale } from '../../../common/locale'
 
 // Images
-import { WelcomeShieldsImage } from 'brave-ui/features/welcome/images'
+import { WelcomeShieldsImage } from '../../../../components/brave_welcome_ui/components/images'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/shieldsBox.tsx]: Same nit for paths

File: components/brave_welcome_ui/containers/screens/themeBox.tsx [hunk 5]
```diff
@@ -5,10 +5,10 @@
 import * as React from 'react'
 
 // Feature-specific components
-import { Content, Title, Paragraph, PrimaryButton } from 'brave-ui/features/welcome'
+import { Content, Title, Paragraph, PrimaryButton } from '../../../../components/brave_welcome_ui/components'
 
 // Images
-import { WelcomeThemeImage } from 'brave-ui/features/welcome/images'
+import { WelcomeThemeImage } from '../../../../components/brave_welcome_ui/components/images'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/themeBox.tsx]: Same nit for paths

File: components/brave_welcome_ui/containers/screens/welcomeBox.tsx [hunk 6]
```diff
@@ -5,13 +5,13 @@
...
 // Feature-specific components
-import { Content, Title, Paragraph, PrimaryButton } from 'brave-ui/features/welcome'
+import { Content, Title, Paragraph, PrimaryButton } from '../../../../components/brave_welcome_ui/components'
 
 // Shared components
 import { ArrowRightIcon } from 'brave-ui/components/icons'
 
 // Images
-import { WelcomeLionImage } from 'brave-ui/features/welcome/images'
+import { WelcomeLionImage } from '../../../../components/brave_welcome_ui/components/images'
```
Comment (petemill) [inline components/brave_welcome_ui/containers/screens/welcomeBox.tsx]: Same nit for paths

File: components/brave_welcome_ui/welcomeUtils.ts [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 import { Dispatch } from 'redux'
 import { getSearchEngineProvidersSuccess, getBrowserProfilesSuccess } from './actions/welcome_actions'
-import { State as ImportBoxState } from '../brave_welcome_ui/components/screens/importBox'
+import { State as ImportBoxState } from '../brave_welcome_ui/containers/screens/importBox'
```
Comment (petemill) [inline components/brave_welcome_ui/welcomeUtils.ts]: nit: path exits UI and then goes back in - can just be `./`

**Comment by cezaraugusto** [comment]
good catch, thanks @petemill. updated

**Pull request merged**